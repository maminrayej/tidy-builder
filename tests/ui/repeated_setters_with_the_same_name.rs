#[derive(tidy_builder::Builder)]
pub struct MyStruct {
    #[builder(each = "args")]
    args: Vec<String>,
}

fn main() {
    let my_struct = MyStruct::builder()
        .args(vec![])
        .build();
}
