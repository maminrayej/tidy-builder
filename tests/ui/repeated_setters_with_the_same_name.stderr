error[E0308]: mismatched types
 --> tests/ui/repeated_setters_with_the_same_name.rs:9:15
  |
9 |         .args(vec![])
  |          ---- ^^^^^^ expected struct `String`, found struct `Vec`
  |          |
  |          arguments to this function are incorrect
  |
  = note: expected struct `String`
             found struct `Vec<_>`
note: associated function defined here
 --> tests/ui/repeated_setters_with_the_same_name.rs:3:5
  |
1 | #[derive(tidy_builder::Builder)]
  |          ---------------------
2 | pub struct MyStruct {
3 |     #[builder(each = "args")]
  |     ^
  = note: this error originates in the macro `vec` (in Nightly builds, run with -Z macro-backtrace for more info)
